testing/unit/rpc/test_transaction.py

```diff
@@ -196,3 +197,87 @@
         await fetch_batch(request_)
 
     assert expected_error_message == error.value.message
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('ids', [[1], ['123'], 123, True, 'e'])
+async def test_list_batches_with_invalid_ids(request_, ids):
+    """
+    Case: list batches with invalid ids.
+    Expect: exception with invalid identifier message is raised.
+    """
+    request_.params = {
+        'ids': ids,
+    }
+
+    with pytest.raises(RpcInvalidParamsError) as error:
+        await list_batches(request_)
+
+    assert 'Invalid id.' == error.value.message
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('start', [1, '1', True])
+async def test_list_batches_with_invalid_start(request_, start):
+    """
+    Case: list batches with invalid start field.
+    Expect: exception with resource not found message is raised.
```

Comment:
Below is `Invalid id` error message. Check all test cases for this, please.

Reply:
Done.